### ISSUE TITLE
Provision wizard in OpenShift should show service identificator on the end

### ISSUE THREAD

**Issue by kornys** [EnMasseProject/enmasse#1219]
Provision wizard in OpenShift should show service identificator on the end (enmasse-{address-space-type}-id). 

current screenshot of last slide in wizard.

![slide](https://user-images.githubusercontent.com/11318335/39984788-9e8377c8-575c-11e8-9530-a630a16d5efc.png)

**Comment by rgodfrey** [comment]
I don't think we (EnMasse) have any control over what is shown in this wizard - this is all OpenShift code
